docs/source/traits_user_manual/custom.rst

```diff
@@ -537,13 +537,12 @@ An application could use this new trait handler to define traits such as the
 following::
 
     # use_custom_th.py --- Example of using a custom TraitHandler
-    from traits.api import HasTraits, Trait, TraitRange
+    from traits.api import HasTraits, Trait, Range
```

Comment:
nit - import order.

Reply:
Darn. I fixed this already but forgot to push. Fix now pushed.